doc/cygnus-ngsi/installation_and_administration_guide/management_interface.md

```diff
@@ -366,6 +367,54 @@ Invalid JSON (Empty field and missing field)
 
 Please observe Cygnus checks if the Json passed in the payload is valid (syntactically and semantically).
 
+[Top](#top)
+
+##<a name="section11"></a>`DELETE /v1/subscriptions`
+
+Deletes a subscription made to Orion with a given subscriptionId and a given ngsi_version. The Json passed in the payload contains the Orion's endpoint details.
```

Comment:
with a given subscriptionId and a given ngsi_version --> given its ID and the NGSI version